### PR TITLE
OSDOCS-4025: Add SR-IOV supported device IDs

### PR DESCRIPTION
**Pull request by jboxman-rh**
This PR adds several additionally supported devices.

- https://issues.redhat.com/browse/OSDOCS-4025

Version(s):
- 4.12

Issue:

Link to docs preview: https://50260--docspreview.netlify.app/openshift-enterprise/latest/networking/hardware_networks/about-sriov.html#supported-devices_about-sriov

Additional information:

### CODE REVIEW

**Comment by wizhaoredhat** [comment]
@bn222 FYI. I think the naming is ok. WDYT.

File: modules/nw-sriov-supported-devices.adoc [hunk 1]
```diff
@@ -87,6 +87,11 @@
 |15b3
 |101f
 
+|Mellanox
+|MT42822 BlueField-2 in ConnectX&#8209;6 NIC mode
```
Comment (bn222) [inline modules/nw-sriov-supported-devices.adoc:91]: make both hyphens non-breaking

**Comment by jboxman-rh** [comment]
@bn222, like this?

```
MT42822 BlueField&#8209;2 in ConnectX&#8209;6 NIC mode
```

Thanks.

**Comment by bn222** [comment]
yes, Looks good

**Review by maxwelldb — APPROVED** [review]

**Pull request merged**